openlibrary/plugins/openlibrary/js/cover_add_manage.js

```diff
@@ -0,0 +1,58 @@
+export default function(){
+    // page may not be loaded via iframe
+    if (parent && parent.closeThrobber) {
+        parent.closeThrobber();
+    }
+    $('#form.addcover-form').submit(function(event) {
+        var file;
+        var url;
+        var coverid;
+        function val(selector) {
+            return $.trim($(selector).val());
+        }
+        function error(message) {
+            $('#errors').show().html(message);
+            event.preventDefault();
+        }
+        file = val('#coverFile');
+        url = val('#imageUrl');
+        coverid = val('#coverid');
+
+        if (file == '' && (url == '' || url == 'http://') && coverid == '') {
```

Comment:
we should use === here instead of ==.
I suspect whoever wrote the initial HTML was probably not familiar with JS.